### ISSUE TITLE
Autosync with owncloud

### ISSUE THREAD

**Issue by svalcken** [MiniKeePass/MiniKeePass#503]
Hello,

can you please add Support for automatical Syncing with owncloud.
Only on my iphone/ipad, i have current no solution für syncing my password database. 

On my MAC, Windows and Linux it give a owncloud Client and the Database can synct with the client, on iPhone and iPad, i miss a sync with owncloud.

Thanks

**Comment by zek0faws** [comment]
Hi,

I don't think it is possible inside the MiniKeePass to sync with owncloud.
The only one way to do taht sync is to export your database to your owncloud client on your iPhone/iPad, an then it'll be sync.

To get the updates from another device, import your database from owncloud to MiniKeePass.

But, unfortunately importing from another app than Dropbox is currently impossible.

**Comment by schnappijedi** [comment]
The native Nextcloud ios app (https://itunes.apple.com/us/app/nextcloud/id1125420102?mt=8) and this app connected via webdav (https://readdle.com/products/documents) can both import and export databases effectively enabling an almost automatic sync.

**Comment by zek0faws** [comment]
@schnappijedi Yes but in that case it's not sync it's webdav. The file is store remotly.

**Comment by schnappijedi** [comment]
Nextcloud/Owncloud clients on all platforms are just fronts for a webdav connection. Any Nextcloud/ Owncloud server can independently be connected via webdav as well.

Both apps above can keep a local ios folder in sync with a webdav folder.

Opening/ importing a database from one of the two apps above in MiniKeePass and then manually uploading the database using either of the two apps above (if made any changes) will keep your database in sync with a Nextcloud/ Owncloud server and is as close to automatic as can get.

**Comment by zek0faws** [comment]
Aaah yes, I thought you were talking about integrate webdav in MiniKeePaas (which can be a nice idea).

But the import from Nxcloud / Owncloud  into MiniKeePass is not working for me.
I don't have the MiniKeePass app into the App menu.

**Comment by schnappijedi** [comment]
What iPhone and ios version are you using?
![img_0098](https://user-images.githubusercontent.com/26072479/30491501-7e231e98-9a0b-11e7-94d1-29c23ec4295c.PNG)
![img_0099](https://user-images.githubusercontent.com/26072479/30491502-7e38ba3c-9a0b-11e7-9ec8-f0a185f6974e.PNG)

**Comment by zek0faws** [comment]
Oups, my bad, I just checked and it's fully working ..

But as you mentioned, it may be a good idea te implement webdav into MiniKeePass, to avoid the import/export

**Comment by schnappijedi** [comment]
Totally agree by the way! Just glad you got a semi auto-sync working.

**Comment by lhurt** [comment]
And it would be even nicer if
- database(s) content would sync with remote
- database changes would be recognized automatically

**Comment by lhurt** [comment]
#91

**Comment by tux1337** [comment]
In the meanwhile Nextcloud is fully integrated in the iOS files app. 
So it should be sufficient if MiniKeePass would open the Keepass file from the original location instead of copy the keepass file. 
In that case the nextcloud app would recognize any change and sync the file. 

I assume this is also valid for owncloud and all the other apps that integrate into files.
